### PR TITLE
Add openQA install testcase using current openQA on current Tumbleweed

### PR DESCRIPTION
**Pull request by okurz**
Test triggered with:
 - INSTALL=1
 - UPDATE=0
 - INSTALL_ONLY=1
 - HDD_1=opensuse-Tumbleweed-x86_64-20160623-gnome@64bit.qcow2

Local verification run: http://lord.arch/tests/1717

Needles needles in https://github.com/os-autoinst/os-autoinst-needles-openQA/pull/2

### CODE REVIEW

**Comment by coolo** [comment]
I merged it as it's surely better than having outdated tests, but we can't deploy this easily as the latest working TW version is unknown. For this purpose it would be better if the openqa tests were part of distri-opensuse

**Comment by coolo** [comment]
But to test openQA it's properly easier to rebase your tests on the predefined 42.1 images.

**Comment by coolo** [comment]
anyway, for a one off: https://openqa.opensuse.org/tests/215887

Clone_job is supposed to be called as geekotest btw

File: lib/utils.pm [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package utils;
+
+use base Exporter;
+use Exporter;
```
Comment (aaannz) [inline lib/utils.pm]: Is it required to inherit from Exporter and then import it?

Comment (coolo) [inline lib/utils.pm]: No, use base is require + push(@ISA) - but this is just a copy of the one in os-autoinst-distri-opensuse

Comment (coolo) [inline lib/utils.pm]: (hi Isa :)

File: tests/update/zypper_up.pm [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
 
 sub run {
-    assert_screen "openqa-desktop", 500;
+    wait_for_desktop;
     send_key "ctrl-alt-f2";
     assert_screen "inst-console";
     type_string "root\n";
-    assert_screen "password-prompt", 10;
+    assert_screen "password-prompt";
     type_string "1\n";
```
Comment (aaannz) [inline tests/update/zypper_up.pm]: I'm confused here. Don't we have select_console for this? And isn't the password different for 13.2 and Tumbleweed base?

Comment (okurz) [inline tests/update/zypper_up.pm]: There is "select_console" but I first have to setup all consoles in lib/susedistribution.pm or clean it up more. Remember, we are in os-autoinst-distri-openQA, not os-autoinst-distri-opensuse.

The password "1" is the password that has been used in the pre-generated 13.2 image that has an already installed openQA instance included.

**Comment by okurz** [comment]
> I merged it as it's surely better than having outdated tests, but we can't deploy this easily as the latest working TW version is unknown. For this purpose it would be better if the openqa tests were part of distri-opensuse
> But to test openQA it's properly easier to rebase your tests on the predefined 42.1 images.


nice. I intend to read dynamically the most recent Tumbleweed version and trigger accordingly. os-autoinst-distri-openQA could be our next to "default" testing suite to find out if os-autoinst and/or openQA are too much intermangled with SUSE distribution testing.

**Pull request merged**